### PR TITLE
Tweaked laser pistols

### LINKED ISSUE #22918: The v29_cheap lacks itemspawns and a recipe
The v29_cheap cannot appear in game at present. I suggest adding it to guns_pistol_improvised with a weight of 5-10.

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Fixes #22918 

Renamed the LACP laser pistol to homemade laser pistol and added spawns to BrightNights mod. Removed old references to fusion ammo from V29 laser pistol and replaced them with size references.

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
Jenkins, rebuild

**Comment by Mecares** [comment]
Maybe add them to Zombie Scientist drops? Building your own laser weapon seems like a fine hobby for the average evil scientist and a great idea if trapped in a lab full of deadly things, while the world goes crazy.

File: data/json/items/gun/ups.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "id": "v29",
     "type": "GUN",
     "name": "V29 laser pistol",
-    "description": "The V29 laser pistol was designed in the mid-21st century and was one of the first firearms to use fusion as its ammunition.  It is larger than most traditional handguns, but displays no recoil whatsoever.",
+    "description": "The V29 laser pistol was designed in the mid-21st century and was one of the first handheld laser weapons.  It is larger than most traditional handguns, but displays no recoil whatsoever.",
```
Comment (Night-Pryanik) [inline data/json/items/gun/ups.json]: This "mid-21st century" should be changed, as we now have agreement that the game take action in the early 21st century.

Comment (BorkBorkGoesTheCode) [reply data/json/items/gun/ups.json]: Might move the description to BrightNights or Icecoon's Arsenal

**Pull request merged**